bin/translations

```diff
@@ -11,3 +11,4 @@ export LANG="${ENCODING:-en_US.UTF-8}"
 set -x
 
 make -C warehouse/locale/ translations
+python bin/translations_json.py
```

Comment:
I _think_ this should probably happen as part of our static build instead: https://github.com/pypi/warehouse/blob/1185eef19866ff464a0db09aed53525bd2b7d46f/Dockerfile#L21-L33

My thinking is that the `.json` files will only get updated when `make translations` is run currently, but that's not something that Weblate runs when it gets a new translation. So we'll have an issue where a PR like https://github.com/pypi/warehouse/pull/16426 lands with new translations, but all the `messages.json` files will be stale, even though the 'Translations' check will pass. We'll have to manually run `make translations` once in a while to bring all the `messages.json` files up to date in the source repo, which is annoying.

I think if we generate these files when we build the rest of our static assets, it shouldn't be necessary to include them in the source repository either, as they'll just be an artifact of the build step before the webpack bundles get built.

Reply:
I've updated `webpack.plugin.localize.js` so that the translation data needed in the js files is generated in the webpack plugin, instead of in the separate `bin/translations_json.py` in `make translations`. This should mean that the `messages.json` files are not needed and the Weblate PRs will generate one js bundle per language.

Reply:
I tested this and it works great. Thank you for addressing it!